tests-arquillian/src/test/java/org/jboss/weld/tests/producer/field/FieldProducerBean.java

```diff
@@ -0,0 +1,11 @@
+package org.jboss.weld.tests.producer.field;
+
+import jakarta.enterprise.context.Dependent;
+import jakarta.enterprise.inject.Produces;
+
+@Dependent
+public class FieldProducerBean {
+
+    @Produces
+    String p = "42";
```

Comment:
I'd try to avoid using JDK types without qualifiers as bean types in the tests. It might happen that the container registers ambiguous beans...

Reply:
Fair enough, I've changed that to a custom type.